src/components/Calculator.js

```diff
@@ -1,34 +1,154 @@
-import React from 'react';
+import React, { useState } from 'react';
 import '../App.css';
 import Button from './Button';
+import calculate from '../logic/Calculate';
+import Quotes from './Quotes';
 
-const Calculator = () => (
-  <div className="main">
-    <div className="calculator">
-      <div className="screen">0</div>
-      <div className="input-container">
-        <Button type="button" text="AC" className="inputs" />
-        <Button type="button" text="+/-" className="inputs" />
-        <Button type="button" text="%" className="inputs" />
-        <Button type="button" text="/" className="inputs-right" />
-        <Button type="button" text="7" className="inputs" />
-        <Button type="button" text="8" className="inputs" />
-        <Button type="button" text="9" className="inputs" />
-        <Button type="button" text="x" className="inputs-right" />
-        <Button type="button" text="4" className="inputs" />
-        <Button type="button" text="5" className="inputs" />
-        <Button type="button" text="6" className="inputs" />
-        <Button type="button" text="-" className="inputs-right" />
-        <Button type="button" text="1" className="inputs" />
-        <Button type="button" text="2" className="inputs" />
-        <Button type="button" text="3" className="inputs" />
-        <Button type="button" text="+" className="inputs-right" />
-        <Button type="button" text="0" className="zero-input" />
-        <Button type="button" text="." className="dot-inputs" />
-        <Button type="button" text="=" className="equal-inputs" />
+const Calculator = () => {
+  const [display, setDisplay] = useState('0');
+
+  const [calculatorData, setCalculatorData] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
+
+  const handleButtonClick = (buttonName) => {
+    const newData = calculate(calculatorData, buttonName);
+    setCalculatorData(newData);
+
+    if (newData.next !== null) {
+      setDisplay(newData.next);
+    } else if (newData.total !== null) {
+      setDisplay(newData.total);
+    } else {
+      setDisplay('0');
+    }
+  };
+  return (
+    <div className="main">
+      <Quotes />
```

Comment:
Hi, thank you for your submission. I noticed that you have imported the `<Quotes>` component into the Calculator component. However, this is not the correct place to import it. The `<Quotes>` component should be imported into the main component (App.js) instead according to project requirements. Please make this change and resubmit your code. Thank you.

<img width="467" alt="image" src="https://github.com/Developer-Kingz/maths-magician/assets/52934128/a56c7987-8365-4d5c-b23a-36436b5fd8af">

See an example below:

```
import './App.css';
import Calculator from './components/Calculator';
import Quote from './components/Quote';

const App = () => (
  <div>
    <Calculator />
    <Quote />
  </div>
);

export default App;
```

Reply:
This change has been done well